index.js

```diff
@@ -0,0 +1,74 @@
+const container = document.querySelector('.container');
+
+// Add book to list
+function addBookToList(book) {
```

Comment:
- [ ] When adding a book to the list, the book persists in the local storage however, when reloading the page, the list of books is not displayed on the page, kindly check how you retrieve data from the Localstorage to fix this issue

![Screenshot_20220927_190212](https://user-images.githubusercontent.com/25227022/192602444-02c8782c-1199-4d1f-807e-9e669df91b9d.png)